src/program.ts

```diff
@@ -1044,7 +1044,7 @@ export class Program extends DiagnosticEmitter {
       // TODO: for (let [alias, name] of globalAliases) {
       for (let _keys = Map_keys(globalAliases), i = 0, k = _keys.length; i < k; ++i) {
         let alias = unchecked(_keys[i]);
-        let name = assert(globalAliases.get(alias));
+        let name = changetype<string>(globalAliases.get(alias))!
```

Comment:
Hmm, this will now result in an `is never null` warning when compiling to Wasm. `npm run asbuild` should show this. Perhaps

```
let name = changetype<string>(globalAliases.get(alias));
assert(name != null);
```

Reply:
Appears that after https://github.com/AssemblyScript/assemblyscript/pull/1177/commits/e8ef003255025732fc4117b13184354b70372029 the `!` leading to the warning is still there. Looking good to me otherwise :)

Reply:
Ah of course the `!`, missed that